### ISSUE TITLE
CW312 target template

### ISSUE THREAD

**Issue by lukaskuzmiak** [newaetech/chipwhisperer-target-cw308t#5]
Would it be possible to eventually get a CW312 target template as well? 🙂

Or if there is some description/something to base in on I'd be happy to volunteer to make a KiCad template version as well.

**Comment by colinoflynn** [comment]
Yes! It's roughly what's in the Altium source for the current ones, I can make mechanical specs. Unlike the CW308 there is a lot more "room" however - so basically you can expand to an unlimited size beyond the edge connector. One of the ideas of the update was making it easier to work with much larger targets, where the CW308T pins started getting in the way!

Will document what we found as best practices here - note that we've tested this pretty extensively with low-cost PCB runs, where you *don't* do hard edge plating and don't bother with the chamfering. It seems to work just fine (I've even done some test runs with HASL on the fingers instead of ENIG, and even that worked fine).
